lib/arena.rb

```diff
@@ -1,2 +1,36 @@
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name
+    @gladiators = []
+  end
+
+  def name
+    @name.capitalize
+  end
+
+  def gladiators
+    @gladiators
```

Comment:
Since we are using `attr_accessor ` and are assigning `@gladiators` in your initialize method, you don't to add an additional method here